01-2024-schedule/week-1.md

```diff
@@ -0,0 +1,182 @@
+---
+description: Building on bitcoin and setting up a bitcoin development environment
+---
+
+# Week 1
+
+## Prerequisites
+
+We would like you to download some source code and attempt to compile it by following its build instructions. You will find yourself using this softwares throughout 
+the course of the course.
```

Comment:
Will it sound better if it were like => You will find yourself using this software throughout the course.?